### DISCUSSION TITLE
Limit how many browser tabs get opened?

### DISCUSSION THREAD

**Discussion by keith92** [jef/streetmerchant#1438] [General]
Is there currently a way to limit how many browser tabs get opened when stock is detected? I keep having issues where an alert gets sent and opens a tab in my browser. I try to place an order but then another tab opens up and takes focus from the one I'm trying to use. I then just get bombarded with new tabs and can't get anything done. I want to limit it to opening one tab per store maybe every five minutes or so. I've looked but haven't been able to locate a config setting or anything for this. Thanks.

**Comment by airsharkie** [comment]
You should be able to modify in .env:

IN_STOCK_WAIT_TIME="300"

From: https://jef.codes/streetmerchant/reference/application/
IN_STOCK_WAIT_TIME | Time to wait between requests to the same link if it has that card in stock. In seconds, default: 0
-- | --

After it sees "in stock" it won't check that link again for 5 minutes. 5minx60sec= "300"

**Reply by keith92** [reply]
How did I miss that? I must have looked past it like 20 times. Thank you.
